pennylane/devices/experimental/custom_device_3_numpydev/python_mps.py

```diff
@@ -91,11 +91,11 @@ def measure_state(cls, state, measurementprocess):
 # TODO: make sure copyrights are not violated
 
 import numpy as np
-from scipy.linalg import svd
+# from scipy.linalg import svd
 # if you get an error message "LinAlgError: SVD did not converge",
 # uncomment the following line. (This requires TeNPy to be installed.)
 #  from tenpy.linalg.svd_robust import svd  # (works like scipy.linalg.svd)
-
+from tenpy.linalg.svd_robust import svd
```

Comment:
This might be annoying for people to test, I needed it for some examples that wouldnt converge otherwise

Reply:
Few suggestions here. If this SVD algorithm (in tenpy) is not difficult, we can make our own version. Otherwise, we should make `tenpy` an optional dependency. It is still strange that `svd` in `scipy` is not robust enough.